package.json

```diff
@@ -23,9 +23,20 @@
     "typescript-sanitizer"
   ],
   "dependencies": {
+    "es6-promise": "^4.0.5",
+    "q": "^1.4.1",
```

Comment:
to use lib user either need to setup target to es6 either install es6-shim. We should not have these libs in dependency